ppr-ui/src/components/tombstone/TombstoneDefault.vue

```diff
@@ -7,11 +7,14 @@
           <span class="tombstone-header">
             <b>{{ header }}</b>
           </span>
-          <v-tooltip top content-class="top-tooltip pa-5" nudge-left="30">
+          <v-tooltip top content-class="top-tooltip pa-5" nudge-left="30" transition="none">
```

Comment:
Does this prop just disable the default transition styling?

Reply:
yes, so the tool-tip doesn't fade in or out

Reply:
cool, thanks!